src/InfrastructureLayer/InfrastructureLayer.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>net7.0</TargetFramework>
+	</PropertyGroup>
+
+	<ItemGroup>
+	  <Compile Remove="Telegram\**" />
+	  <EmbeddedResource Remove="Telegram\**" />
+	  <None Remove="Telegram\**" />
+	</ItemGroup>
```

Comment:
Looks like you don't need Telegram folder at all

Reply:
deleted